### PR TITLE
Fix locale

### LINKED ISSUE #32936: The Chinese translation prompt message of sync fork is wrong
### Description

image
image

### Gitea Version

1.23-rc0

### Can you reproduce the bug on the Gitea demo site?

Yes

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

./gitea web

### Database

None

### PR DESCRIPTION
**Pull request by wxiaoguang**
Fix #32936

### CODE REVIEW

**Review by delvh — APPROVED** [review]

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -1946,8 +1946,8 @@ pulls.delete.title = Delete this pull request?
 pulls.delete.text = Do you really want to delete this pull request? (This will permanently remove all content. Consider closing it instead, if you intend to keep it archived)
 
 pulls.recently_pushed_new_branches = You pushed on branch <strong>%[1]s</strong> %[2]s
-pulls.upstream_diverging_prompt_behind_1 = This branch is %d commit behind %s
-pulls.upstream_diverging_prompt_behind_n = This branch is %d commits behind %s
+pulls.upstream_diverging_prompt_behind_1 = This branch is %[1]d commit behind %[2]s
```
Comment (hiifong) [inline options/locale/locale_en-US.ini:1949]: ~~No modification is required here~~

Comment (hiifong) [reply options/locale/locale_en-US.ini:1949]: What we need to modify should be `options/locale/locale_zh-CN.ini`
```ini
pulls.upstream_diverging_prompt_behind_1=该分支落后于 %[2]s %[1]d 个提交
pulls.upstream_diverging_prompt_behind_n=该分支落后于 %[2]s %[1]d 个提交
```

Comment (wxiaoguang) [reply options/locale/locale_en-US.ini:1949]: No, it can't be done on main branch. The backport needs to manually fix it.

Comment (wxiaoguang) [reply options/locale/locale_en-US.ini:1949]: Maybe some maintainers should document how the Crowdin works @techknowlogick @lunny

Comment (hiifong) [reply options/locale/locale_en-US.ini:1949]: Sorry, I don't understand this workflow yet

Comment (wxiaoguang) [reply options/locale/locale_en-US.ini:1949]: Yup, I do not quite understand either, so I mentioned techknowlogick & lunny

**Review by lunny — APPROVED** [review]

File: options/locale/locale_en-US.ini (hunk 1 above)
Comment (lunny) [reply options/locale/locale_en-US.ini:1949]: Crowdin sync will only happen on the main branch. For other branches, we have to backport manually.

**Pull request merged**